### PR TITLE
Translate the builder readme to Portuguese

### PR DESCRIPTION
**Pull request by ricardo-ramos-moura**
I'll translate the readme file of the builder design pattern to Portuguese and place it under localization.pt.builder.

### CODE REVIEW

File: localization/pt/builder/README.md [hunk 1]
```diff
@@ -0,0 +1,155 @@
+---
+layout: pattern
+title: Builder
+folder: builder
+permalink: /patterns/builder/
+categories: Creational
+language: pt
+tags:
+ - Gang of Four
+---
```
Comment (iluwatar) [inline localization/pt/builder/README.md:10]: Please update the yaml frontmatter according to the updated requirements at https://github.com/iluwatar/java-design-patterns/wiki/01.-How-to-contribute. Basically you can just clone them from https://raw.githubusercontent.com/iluwatar/java-design-patterns/master/builder/README.md

**Review by iluwatar — CHANGES_REQUESTED** [review]

File: localization/pt/builder/README.md (hunk 1 above)
Comment (ricardo-ramos-moura) [reply localization/pt/builder/README.md]: I updated the yaml frontmatter as requested. Thanks @iluwatar

Comment (ricardo-ramos-moura) [reply localization/pt/builder/README.md]: @iluwatar could you please check this PR. Thank you

File: localization/pt/builder/README.md [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+título: Builder
+categoria: Criacional
+idioma: pt
+tags:
+- Gang of Four
+---
```
Comment (iluwatar) [inline localization/pt/builder/README.md:7]: This part should not be translated

**Review by iluwatar — CHANGES_REQUESTED** [review]

**Review by iluwatar — APPROVED** [review]

**Comment by iluwatar** [comment]
Looks good! Thank you for the contribution 🎉

@all-contributors please add @ricardo-ramos-moura for translation

**Comment by iluwatar** [comment]
Part of #2348

**Pull request merged**